2 - RTB/Verbali/Interni/23-11-13/23-11-13.typ

```diff
@@ -0,0 +1,84 @@
+#import "template.typ": *
+
+// Take a look at the file `template.typ` in the file panel
+// to customize this template and discover how it works.
+#show: project.with(
+  title: "Verbale **13/11/23**",
+  subTitle: "***Subtitle***",
+  docType: "verbale",        
+  partecipants: (
+    (name: "Banzato Alessio", role: "Verificatore"),
+    (name: "Carraro Riccardo", role: "Verificatore"),
+    (name: "Gardin Giovanni",  role: "Redattore"),
+    (name: "Nardo Silvio",   role: "Redattore"),
+    (name: "Oseliero Antonio",   role: "Amministratore"),
+    (name: "Todesco Mattia",  role: "Verificatore"),
+    (name: "Zaccone Rosario",  role: "Amministratore"),
+    //(name: "Pippo",  role: "Proponente"),
+  ),
+  oraInizio: "15:30",
+  oraFine:  "17:30",
+  firmeEsterne: "///inserire firma esterna///"   //Questa variabile va aggiunta solo se è un documento esterno, qua i partecipanti esterni andranno a firmarsi.
+)
+
+#set text(font:"New Computer Modern", size: 1.3em)
+#set align(left)
+
+
+
+= Ordine del giorno
+  - Aggiornamento del gruppo in merito alle nuove automazioni adottate;
+  - Discussione in merito al precedente sprint;
```

Comment:
cambierei in "Retrospettiva dello sprint 1"

Reply:
noi adesso siamo allo sprint 3, quindi quello era lo sprint 2; comunque cambio senza problemi